### DISCUSSION TITLE
MySQL or PostgreSQL as backend database for airflow metadata?

### DISCUSSION THREAD

**Discussion by hailindu** [apache/airflow#22405] [Q&A]
Hello everyone,

MySQL vs. PostgreSQL, which one you would prefer to use as the backend database to store for airflow metadata? I am planning to create one but not sure which one I should use, and googling didn't seem to help.

And pro and con for these two? 

Suggestions or recommendations would be largely appreciated. Many thanks!!

**Comment by potiuk** [comment]
Postgres. Full Stop.

**Reply by hailindu** [reply]
hmmm, by any chance would you mind explaining a little bit more about what Full Stop means?

**Reply by potiuk** [reply]
Just search our issues and look for MySQL. Then look for Postgres. You will likely see yourself. Simply Postgres is rock-solid and MySQL has multiple problems with reliability

**Reply by hailindu** [reply]
got you, thank you so much!
